downloaded from
http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/app-misc/empty/files/empty-respect-LDFLAGS.patch?view=markup

Reported on the Gentoo bug tracker
https://bugs.gentoo.org/show_bug.cgi?id=429664

and reported upstream
https://sourceforge.net/tracker/?func=detail&aid=3554236&group_id=136798&atid=736886

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>

--- empty-0.6.23c/Makefile.orig
+++ empty-0.6.23c/Makefile
@@ -22,7 +22,7 @@
 UnixWare-gcc OpenUNIX-gcc HP-UX-gcc SunOS-gcc
 
 all:
-	${CC} ${CFLAGS} -Wall ${LIBS} -o empty empty.c
+	${CC} ${CFLAGS} ${LDFLAGS} empty.c ${LIBS} -o empty
 
 FreeBSD:	all
 NetBSD:		all
